lib/shopify-cli/commands/connect.rb

```diff
@@ -0,0 +1,79 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Commands
+    class Connect < ShopifyCli::Command
+
+      def call(*)
+        if Project.at(Dir.pwd)
+          @ctx.puts "{{yellow:! Don't use}} {{cyan:connect}} {{yellow:for production apps}}"
+          org_id = get_organization
+          app = get_app(org_id)
+          shop = get_shop
+          write_env(app, shop)
+          @ctx.puts "{{v}} Project now connected to {{green:app_name}}"
+          @ctx.puts "{{*}} Run {{cyan:shopify serve}} to start a local development server"
+        end
+      end
+
+      def get_organization
+        orgs = Helpers::Organizations.fetch_all(@ctx)
```

Comment:
You should just add the app query stuff to this query so you only need to do one query.